A3-Antistasi/resourcecheck.sqf

```diff
@@ -157,7 +157,20 @@ while {true} do
 			};
 		};
 	sleep 3;
-	if ((count antennasDead > 0) and (not(["REP"] call BIS_fnc_taskExists))) then
+	_wreckedAntennas = false;
+	_wc = count antennasDead;
+	if (_wc > 0) then {_wreckedAntennas = true};
+	_smallWar = false;
+	_bigWar = false;
+	if ((tierWar<=3)and{not(["REP"] call BIS_fnc_taskExists)}) then
+	{_smallWar = true};
+	if ((tierWar>3)and{(count(["REP"] call BIS_fnc_taskExists)>1)}) then
```

Comment:
This condition can never be satisfied - there's at most 1 REP mission, the task system won't let you have more

Reply:
should be <2, will fix. and probably makes the other part more clear